sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -250,22 +233,24 @@ azure:
                - <Web-API-B-app-id-url>/File.Read
 ```
 
-#### Support the use of `@RegisteredOAuth2AuthorizedClient` to get `OAuth2AuthorizedClient`:
-```java
- @PreAuthorize("hasAuthority('SCOPE_Obo.Graph.Read')")
- @GetMapping("call-graph")
- public String callGraph(@RegisteredOAuth2AuthorizedClient("graph") OAuth2AuthorizedClient graph) {
-     return callMicrosoftGraphMeEndpoint(graph);
- }
- 
- @PreAuthorize("hasAuthority('SCOPE_Obo.File.Read')")
- @GetMapping("call-custom")
- public String callCustom(@RegisteredOAuth2AuthorizedClient("custom") OAuth2AuthorizedClient custom) {
-     return callCustomEndpoint(custom);
- }
-```
+#### Standalone resource server
+Only as a Resource Server, no further access to other resources protected by Azure AD.
+![Standalone resource server usage](resource/add-based-standalone-resource-server.png)
 
-### [Web API] (Deprecated) Authenticate in web API by a filter
+* Access restricted resources of Resource Server.
+* Validate access token.
+* Return secured data.
+
+#### Resource server access other resources
+Resource server accesses other resource servers which are protected by Azure AD.
+![Resource Server Access Other Resources](resource/add-based-resource-server-access-other-resources.png)
+
+* Access restricted resources related to Graph and Custom resources through resource server.
+* Auto On-Behalf-Of to request an access token for other clients.
+* Use each client's access token to request restricted resource.
+* Return secured data.
+
+### (Deprecated) Authenticate in web API by a filter
```

Comment:
Delete the 2 deprecated item.

Reply:
Thanks for your suggestions. I just updated it. Trouble you to review again. 🙏